docs/pages/index.mdx

```diff
@@ -1,25 +1,72 @@
+
+The need to scale was evident and the Ethereum community answered the call. While the expansion of the L2 network has been widely successful in bringing down transaction costs, enabling experimentation, and facilitating the entry of new users to the Ethereum ecosystem, it has not been without challenges of its own. One of these challenges has been interoperability. 
+
+Different blockchains, such as Ethereum and the network of L2s and rollups, do not natively communicate with each other. 
+
+Crosschain messaging protocols enable interoperability between these different protocols by allowing smart contracts on one blockchain to interact with contracts and assets on another through the transfer of data. 
+
+One popular use case of messaging protocols is bridges. These are often used to transfer assets, tokens, and NFTs across blockchain networks for use in DeFi protocols, on-chain games, and socialFi.
+
+There are currently a number of protocols which enable crosschain communication. While this growing network of protocols has made crosschain activities a reality, the development of multiple different technical solutions and methodologies has come at the cost of broader compostability and a unified developer and user experience. 
+
+## ERC-7786 at a Glance
+
+The objective of ERC-7786 is to provide a standard interface to send and receive crosschain messages between smart contracts. 
+
+
+It allows app developers to use a common framework, minimizing lock-in to specific crosschain messaging technologies. Protocol-specific features can remain available under this common framework through non-standard attributes, without breaking interoperability of the core functionality of simple cross-chain messaging.
+
+Messaging protocols that do not natively implement this standard can be brought into the framework via adapters, ensuring broad compatibility with existing solutions, while reducing challenges and risks associated with lock-in to specific technologies and bridges. 
+
+## Benefits of ERC-7786
+
+ - **Enhances Ethereum Scalability** by providing a single universal interface for crosschain messaging, uniting the Ethereum and L2 network through increased interoperability. 
+
+ - **Increases User Choice:** reducing vendor lock-in and improving portability. 
+
+ - **Improves the Developer Experience:** Protocol-specific features can be implemented as attributes, making the standard flexible while maintaining a shared core of functionality.
+
+ - **Design Flexibility:** Some protocols require additional parameters or actions to complete message delivery. ERC-7786 accommodates these requirements via post-processing or attributes. 
+
+ - **Enhanced Security:** The standard ensures that cross-chain messages are verified and processed securely, protecting assets and permissions.
```

Comment:
This is not a benefit of ERC-7786 itself since it depends on the underlying protocol. I would remove it from this list.